### PR TITLE
 484916 add ticket number to commit message git hook

### PR DESCRIPTION
**Pull request by feedmypixel**

### CODE REVIEW

File: .husky/prepare-commit-msg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set +e
+
+# Extract the current branch nam
+BRANCH_NAME=$(git symbolic-ref --short HEAD)
+
+# Define the regex pattern for the ticket ID
+TICKET_ID_REGEX="^[0-9]+"
```
Comment (flurdy) [inline .husky/prepare-commit-msg:8]: `^[0-9]{5,7}+` may be more certain it is a ticket id not just `2nd-attempt` false positives

Comment (flurdy) [reply .husky/prepare-commit-msg:8]: May delimit it with `-_` as well?

**Review by flurdy — APPROVED** [review]

File: .husky/prepare-commit-msg (hunk 1 above)
Comment (feedmypixel) [reply .husky/prepare-commit-msg]: Needs to `^[0-9]{5,8}`

**Comment by feedmypixel** [comment]
Ended up removing this as it was messing with my `fixups`

**Pull request merged**